src/ts/src/common/styles/typography.scss

```diff
@@ -0,0 +1,147 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,600;0,700;0,800;1,300;1,400;1,600;1,700;1,800&display=swap");
+
+// font families
+$font-family-primary: "Open Sans", "Helvetica Neue", Arial, Helvetica,
```

Comment:
is it ok that only some of these in quotes?

Reply:
Yeah, they're in quotes because there is a space in their name.